webapp/components/ContributionForm/index.vue

```diff
@@ -144,6 +165,9 @@ export default {
         this.form.languageOptions.push({ label: locale.name, value: locale.code })
       })
     },
+    addTeaserImage(file) {
+      this.form.teaserImage = file
+    },
   },
   apollo: {
```

Comment:
> [<img alt="Kachulio1" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Kachulio1) **Authored by [Kachulio1](https://github.com/Kachulio1)**
_<time datetime="2019-06-24T17:57:30Z" title="Monday, June 24th 2019, 7:57:30 pm +02:00">Jun 24, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -89,8 +128,15 @@ export default {
         this.form.content = contribution.content
         this.form.title = contribution.title
         this.form.language = this.locale
+        this.form.teaserImage = contribution.imageUpload
       },
     },
+    error() {
+      let that = this
```


have you tried to use an arrow function

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-06-24T19:42:39Z" title="Monday, June 24th 2019, 9:42:39 pm +02:00">Jun 24, 2019</time>_

---


I did, it broke the functionality :worried: